src/content/tutorials/connectivity/gateway-api/_index.md

```diff
@@ -0,0 +1,424 @@
+  - What are the Gateway API components and how do they work together?
+  - How do I migrate from Ingress to Gateway API?
+last_review_date: 2025-10-15
+---
+
+The Kubernetes Gateway API is the next-generation standard for managing ingress traffic in Kubernetes clusters. It provides a more expressive, extensible, and role-oriented approach to traffic management compared to traditional Ingress resources. Giant Swarm supports Gateway API through Envoy Gateway, providing advanced load balancing, traffic routing, and API gateway capabilities.
+
+This guide explains how to set up and use Gateway API with Envoy Gateway in Giant Swarm workload clusters.
+
+## Overview
+
+Gateway API introduces several key concepts that provide more flexibility and control over traffic management:
+
+### Gateway API vs Ingress
+
+| Feature | Ingress | Gateway API |
+|---------|---------|-------------|
+| **Role separation** | Single resource | Separate resources for infrastructure (Gateway) and routing (HTTPRoute) |
+| **Protocol support** | HTTP/HTTPS only | HTTP/HTTPS, TCP, UDP, TLS |
+| **Extensibility** | Limited | Highly extensible with custom resources |
+| **Traffic policies** | Basic | Advanced (retries, timeouts, load balancing) |
+| **Multi-tenancy** | Limited | Built-in role-based access control |
+
+### Key Components
+
+The Gateway API consists of three main components available in Giant Swarm:
+
+1. **[Gateway API CRDs](https://github.com/giantswarm/gateway-api-crds-app)**: Core custom resource definitions for Gateway API
+2. **[Envoy Gateway](https://github.com/giantswarm/envoy-gateway-app)**: The gateway implementation based on Envoy Proxy
+3. **[Gateway API Config](https://github.com/giantswarm/gateway-api-config-app)**: Default configuration for quick setup
+
+### Gateway API Resources
+
+- **GatewayClass**: Defines the type of gateway (managed by platform team)
+- **Gateway**: Configures load balancer and listeners (managed by platform team)
+- **HTTPRoute**: Defines HTTP routing rules (managed by application teams)
+- **TCPRoute/UDPRoute**: Defines TCP/UDP routing rules
+- **ReferenceGrant**: Enables cross-namespace references
+
+## Prerequisites
+
+Before setting up Gateway API, ensure you have:
+
+- A Giant Swarm workload cluster
+- `kubectl` configured to access your workload cluster
+- Access to the Giant Swarm platform API for app installation
+- Basic understanding of Kubernetes networking concepts
+
+## Installation
+
+Gateway API support is provided through three apps that work together. You can install them individually or use the Gateway API Bundle for simplified deployment.
+
+### Option 1: Install Gateway API Bundle (Recommended)
+
+The easiest way to get started is using the Gateway API Bundle, which installs all required components:
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-bundle \
+  --target-namespace=envoy-gateway-system \
```

Comment:
The target-namespace of the bundle is the org namespace (because it should land in the MC).

Also, the label for app-operator is required